sdk/communication/azure-mgmt-communication/setup.py

```diff
@@ -82,7 +82,7 @@
     ]),
     install_requires=[
         'msrest>=0.5.0',
-        'msrestazure>=0.4.32,<2.0.0',
+        'azure-mgmt-core>=1.2.0,<2.0.0',
```

Comment:
Is this auto generated or manual updated?

Remove duplicated  `'azure-mgmt-core>=1.2.0,<2.0.0',`
just like below,

```
install_requires=[
        'msrest>=0.5.0',
        'azure-common~=1.1',
        'azure-mgmt-core>=1.2.0,<2.0.0'
    ]

```

Reply:
duplicated 'azure-mgmt-core>=1.2.0,<2.0.0' should be removed. It is problem of auto-releaase. Another 'azure-mgmt-core'  is added by others according to commit history